junit5-extension/src/main/java/eu/drus/jpa/unit/api/JpaUnit.java

```diff
@@ -110,7 +64,17 @@ public ExecutionContext getContext() {
 
             @Override
             public boolean hasErrors() {
-                return considerExceptions ? ((TestExtensionContext) context).getTestException().isPresent() : false;
+                return considerExceptions ? context.getExecutionException().isPresent() : false;
+            }
+
+            @Override
+            public FeatureResolver getFeatureResolver() {
+                return FeatureResolver.newFeatureResolver(getTestMethod(), getTestClass()).build();
+            }
+
+            @Override
+            public Object getTestInstance() {
+                return context.getTestInstance().get();
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Call "Optional#isPresent()" before accessing the value. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS3655)